BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -128,3 +128,10 @@
     {% endspaceless %}
 {% endblock form_help %}
 
+{% block widget_attributes %}
+{% spaceless %}
+        id="{{ id }}" name="{{ full_name }}"{% if read_only %} readonly="readonly"{% endif %}{% if disabled %} disabled="disabled"{% endif %}{% if required %} required="required"{% endif %}{% if max_length %} maxlength="{{ max_length }}"{% endif %}{% if pattern %} pattern="{{ pattern }}"{% endif %}
```

Comment:
could you put it in multiline ?

You could use {%- and -%} to suppress the white spaces